server/kit/kitserver.go

```diff
@@ -89,27 +92,26 @@ func NewServer(svc Service) *Server {
 		propr propagation.HTTPFormat
 	)
 
-	projectID := googleProjectID()
-	var svcName, svcVersion string
-	if isGAE() {
-		_, svcName, svcVersion = getGAEInfo()
-	} else if n, v := os.Getenv("SERVICE_NAME"), os.Getenv("SERVICE_VERSION"); n != "" {
-		svcName, svcVersion = n, v
+	projectID, svcName, svcVersion := observe.GetServiceInfo()
+	onErr := func(err error) {
+		lg.Log("error", err, "message", "tracing client encountered an error")
```

Comment:
Since this exporter is used for both tracing and metrics, mind making this a bit more generic to avoid confusion?